rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    book.add_rental(self)
+    @person = person
+    person.add_rental(self)
+  end
```

Comment:
- [ ] The rental class seems to have an issue with accepting parameters. I got an error that I have given it one parameter instead of two. This is an issue to do with associations, kindly take a look and fix the issue for the project to be approved.

![rental](https://user-images.githubusercontent.com/53124548/229286805-a696f373-4c8b-42ee-8a18-3ee8f940c93e.png)

Reply:
@elisha2kyakpo1 
![image](https://user-images.githubusercontent.com/41564330/229287253-22cd2b8d-58dd-43b3-b334-185f1d6775cc.png)